common/tls.go

```diff
@@ -13,16 +13,29 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
 )
 
-// Redis TLS file settings
+// TLS file settings
 const (
+	// General settings
 	TlsCertOutputDir    = "/tmp/edgex/secrets/ca"
 	CaKeyFileName       = "ca.key"
 	CaCertFileName      = "ca.crt"
-	RedisKeyFileName    = "redis.key"
-	RedisCsrFileName    = "redis.csr"
-	RedisCertFileName   = "redis.crt"
 	OpensslConfFileName = "openssl.conf"
 	RsaKySize           = "4096"
+
+	// Redis specific settings
+	RedisKeyFileName  = "redis.key"
+	RedisCsrFileName  = "redis.csr"
+	RedisCertFileName = "redis.crt"
+
+	// MQTT specific settings
+	MqttServerCertOutputDir = "/tmp/edgex/secrets/ca/mqtt"
```

Comment:
why do you need the Dir?  They should be stored in the named volume

Reply:
Removed.